### DISCUSSION TITLE
Feedback/suggestions

### DISCUSSION THREAD

**Discussion by brndnmtthws** [jpanther/congo#303] [General]
Thanks for making this theme, you've done a very nice job. I have a bit of feedback and suggestions, so I thought I'd share them here.

Here's my website where I'm using it:

* [Website](https://brndn.io/)
* [GitHub repo](https://github.com/brndnmtthws/brndn-io)

I changed very little in the layout, but these are the things worth noting:

* The `/index.json` page returns everything, including tags, which messes up the search a bit. [Only including regular pages fixes that](https://github.com/brndnmtthws/brndn-io/blob/master/layouts/_default/index.json#L2).
* There's a bug in the search button layout when switching between screen sizes, I fixed it here by removing the `sm` variant: https://github.com/brndnmtthws/brndn-io/blob/master/layouts/partials/header.html#L50
* I added a "related posts" section to the article pagination partial, though it can be a separate partial: https://github.com/brndnmtthws/brndn-io/blob/master/layouts/partials/article-pagination.html#L74-L99

Also, I think the text could be a shade lighter on dark themes. [This](https://github.com/jpanther/congo/blob/6cdedf6ad443f294ada98957ce3399a3b10c91e5/tailwind.config.js#L70) could be changed to:

```js
            "--tw-prose-invert-body": theme("colors.neutral.200 / 1"),
```

And I think it would look a lot better (i.e., from `colors.neutral.200` to `colors.neutral.300`).

As you already know, it's a bit annoying to update the compiled CSS for Taliwinds, so it would be good to figure out a better way to handle that. Just food for thought.

Cheers, and thanks again for the great work!

**Comment by MikiVanousek** [comment]
I like your share buttons. Did you make these yourself, or did I miss those in the docs?

**Reply by brndnmtthws** [reply]
Those are included: https://github.com/brndnmtthws/brndn-io/blob/1ef1083252329112ca4c1f5060fde0bd4c470dbc/config/_default/params.toml#L48-L55
